concent_api/core/exceptions.py

```diff
@@ -1,3 +1,7 @@
 
 class UnexpectedResponse(Exception):
     pass
+
+
+class ConcentInShutdownMode(Exception):
+    pass
```

Comment:
Why ? Currently we handle each of them separately. 

What would be nice, is if we add custom error classes for everything, and if they include error code inside them. This way we will have exceptions and error codes in single place.

Reply:
Yeah, combining error codes and exceptions is a good idea.

As to the base class - I was just asking. I'm not saying we have to have one. But I think that it might be useful to be able to selectively catch (or not catch) all the non-built-in stuff. If you think we don't need one, let's leave it as is.

Reply:
We might need it, but not yet. Currently our exception handling is doing ok, but in the future I see possibility that we will need it.